lib/PuppeteerSharp/Launcher.cs

```diff
@@ -202,32 +274,61 @@ private void CleanUp()
             _chromeProcess?.RemoveExitedEvent();
         }
 
+        private async Task AfterProcessExit()
+        {
+            if (!IsChromeClosed)
```

Comment:
maybe rewrite as
```cs
if(IsChromeClosed) return;
...
```
to lower the nesting

Reply:
Will do but with braces.

Reply:
Done